testsuite/integration/rts/src/test/java/org/wildfly/test/extension/rts/ParticipantTestCase.java

```diff
@@ -68,7 +67,7 @@
     public static WebArchive getDeployment() {
         return AbstractTestCase.getDeployment()
                 .addClasses(LoggingParticipant.class, AbstractTestCase.class, TestSuiteEnvironment.class)
-                .addAsWebInfResource(new File("../test-classes", "web.xml"), "web.xml")
+                .addAsWebInfResource(InboundBridgeTestCase.class.getClassLoader().getResource("web.xml"),"web.xml")
```

Comment:
ParticipantTestCase.class

seems like copy&paste from InboundBridgeTestCase.java

Reply:
yeah it is same resource, but it should be fixed.